requirements

```diff
@@ -0,0 +1,2 @@
+django
```

Comment:
You should use a specific version, so other developera will have the same experience 
Also you can add requirements_dev.text that contain requirements related to development and testing
Such as black, pre-commit, pytest, ....